python/dgl/heterograph.py

```diff
@@ -6039,6 +6039,60 @@ def int(self):
         """
         return self.astype(F.int32)
 
+    def half_(self):
```

Comment:
Two suggestions:
* Make them out-placed to comply with PyTorch's convention.
* Make these APIs more distinguished with the APIs for converting index types of graph structure.

Reply:
This has been removed in favor of `dgl.transforms.to_half()/dgl.transforms.to_float()`